### DISCUSSION TITLE
light table / onion skin ideas

### DISCUSSION THREAD

**Discussion by mshoek** [keiwando/toonsquid#382] [Ideas]
First of all, I must say I really like ToonSquid, especially since the latest update. It's the best animation app for the iPad. Thank you Keiwando! <3

But despite this, I still have a few suggestions/requests. I try to post them thematically in different discussions. I hope some of it can make it into the app someday.

LIGHT TABLE / ONION SKIN

**a. More onion skin steps before-after (maybe 8-10)**

**b. Option for onion-skinning between bookmarks/notes**

**c. In addition to the sliders, some buttons for visibility**

**d. Out of pegs**
I started a discussion about this earlier and said that the transform tool can be a substitute for this. But in reality, it's too much of a hassle to arrange the drawings. It would really be great if it could be a built-in feature!

I attach a video of the TvPaint light table, with the wanted features. 

https://github.com/keiwando/toonsquid/assets/118011378/12227015-5554-4dd9-9a57-972861fa44c9

**Comment by DWrightAnimation** [comment]
I want all those features, it could help speed up my workflow. It'd also be nice to have a forward/backward (z-axis) option.
